datasets/cbt/README.md

```diff
@@ -0,0 +1,334 @@
+---
+annotations_creators:
+- machine-generated
+language_creators:
+- found
+languages:
+- en
+licenses:
+- gfdl-1.3
+multilinguality:
+- monolingual
+size_categories:
+  CN:
+  - 100K<n<1M
+  NE:
+  - 100K<n<1M
+  P:
+  - 100K<n<1M
+  V:
+  - 100K<n<1M
+  raw:
+  - n<1K
+source_datasets:
+- original
+task_categories:
+  CN:
+  - question-answering
+  NE:
+  - question-answering
+  P:
+  - question-answering
+  V:
+  - question-answering
+  raw:
+  - other
+task_ids:
+  CN:
+  - multiple-choice-qa
+  NE:
+  - multiple-choice-qa
+  P:
+  - multiple-choice-qa
+  V:
+  - multiple-choice-qa
+  raw:
+  - other-other-raw-dataset
+---
+
+# Dataset Card for CBT
+
+## Table of Contents
+- [Dataset Card for CBT](#dataset-card-for-dataset-name)
+  - [Table of Contents](#table-of-contents)
+  - [Dataset Description](#dataset-description)
+    - [Dataset Summary](#dataset-summary)
+    - [Supported Tasks and Leaderboards](#supported-tasks-and-leaderboards)
+    - [Languages](#languages)
+  - [Dataset Structure](#dataset-structure)
+    - [Data Instances](#data-instances)
+    - [Data Fields](#data-fields)
+    - [Data Splits](#data-splits)
+  - [Dataset Creation](#dataset-creation)
+    - [Curation Rationale](#curation-rationale)
+    - [Source Data](#source-data)
+      - [Initial Data Collection and Normalization](#initial-data-collection-and-normalization)
+      - [Who are the source language producers?](#who-are-the-source-language-producers)
+    - [Annotations](#annotations)
+      - [Annotation process](#annotation-process)
+      - [Who are the annotators?](#who-are-the-annotators)
+    - [Personal and Sensitive Information](#personal-and-sensitive-information)
+  - [Considerations for Using the Data](#considerations-for-using-the-data)
+    - [Social Impact of Dataset](#social-impact-of-dataset)
+    - [Discussion of Biases](#discussion-of-biases)
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+    - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:**[The bAbI project](https://research.fb.com/downloads/babi/)
+- **Repository:**
+- **Paper:** [arXiv Paper](https://arxiv.org/pdf/1511.02301.pdf)
+- **Leaderboard:**
+- **Point of Contact:** [Felix Hill](mailto:felix.hill@cl.cam.ac.uk) or [Antoine Bordes](mailto:abordes@fb.com).
+
+### Dataset Summary
+
+[More Information Needed]
```

Comment:
Can you add a high level description of the dataset here ?

Can you also explain the differences between the different configurations ?